vllm/model_executor/models/phi3v.py

```diff
@@ -511,10 +512,19 @@ def __init__(self,
         self.sampler = Sampler()
 
     def _validate_image_sizes(self, data: torch.Tensor) -> torch.Tensor:
-        if list(data.shape[1:]) != [2]:
-            raise ValueError(
-                f"The expected shape of image sizes is batch dimension plus "
-                f"{[2]}. You supplied {tuple(data.shape)}.")
+        expected_dims = (2, )
+
+        def _validate_shape(d: torch.Tensor):
```

Comment:
Although this is more complex, the error message becomes consistent with the one for `pixel_values`.